src/Stepic/CalcIntegral.java

```diff
@@ -0,0 +1,60 @@
+package Stepic;
+//Implement a method that numerically integrates a given function over a given interval using the left rectangle formula.
+import java.util.function.DoubleUnaryOperator;
+
+public class CalcIntegral {
+    public static void main(String[] args) {
+        System.out.println(integrateLeft(x -> 1, 0, 10));//10.0
```

Comment:
Can you explain this expression?          x -> 1
1. How does it call?
2. How does it work?

Reply:
1) this is a lambda expression
2) as far as I understand this task of the function, in our case it can be written as a function y = 1 (straight line along the x-axis)